win32ss/user/ntuser/menu.c

```diff
@@ -2992,10 +2992,8 @@ static BOOL FASTCALL MENU_ShowPopup(PWND pwndOwner, PMENU menu, UINT id, UINT fl
     /* We are off the right side of the screen */
     if (x + width > monitor->rcMonitor.right)
```

Comment:
For consistency:
```suggestion
    /* We are off the right edge of screen */
    if (x + width - 1 > monitor->rcMonitor.right)
```

Reply:
Something to check, but since `rcMonitor` is a RECT, someone please check this: https://devblogs.microsoft.com/oldnewthing/20040218-00/?p=40563

Reply:
I concur: this function should explicitly document, for example, whether `x`, `y` and `monitor->rcMonitor.*` are included or excluded to draw the menu over, and code and comments should match that.

Reply:
I have not attempted to document some of the details of the screen size and x and y values. Thanks.

Reply:
In principle this is common knowledge in Win32 programming.